src/mapeo-manager.js

```diff
@@ -365,6 +371,14 @@ export class MapeoManager {
       projectInfo,
     })
 
+    // 5. Write device info into project
+    const deviceInfo = await this.getDeviceInfo()
+
+    if (deviceInfo.name) {
+      const project = await this.getProject(projectPublicId)
+      await project.$setOwnDeviceInfo({ name: deviceInfo.name })
+    }
```

Comment:
same concern as previously stated regarding needing to call `manager.setDeviceInfo()` first